docs/getting-started/offline-endpoint.asciidoc

```diff
@@ -22,6 +22,8 @@ Complete these steps:
 
 Set up and configure an HTTP reverse proxy to forward requests to https://artifacts.security.elastic.co and include response headers from the elastic.co server when proxying.
 
+IMPORTANT: The entity tag (Etag) header is a mandatory HTTP response header that you _must_ set in your server configuration file. {elastic-endpoint} uses the Etag header to determine whether your global artifacts have been updated since they were last downloaded. If your server configuration file does not contain an ETag header, {elastic-endpoint} won't download new artifacts when they're available.
```

Comment:
Nit for consideration (_Nitsideration_?): Use monospace for `Etag`, following Mozilla's [example](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/ETag)? [Our guidelines](https://docs.elastic.dev/content-design/emphasis) don't mention HTTP headers specifically, but they're similar to other things we format with monospace, like field names or code in general.

Reply:
yeah, that works- I have no objections to using monospace when styling headers.

```suggestion
IMPORTANT: The entity tag (`Etag`) header is a mandatory HTTP response header that you _must_ set in your server configuration file. {elastic-endpoint} uses the `Etag` header to determine whether your global artifacts have been updated since they were last downloaded. If your server configuration file does not contain an `ETag` header, {elastic-endpoint} won't download new artifacts when they're available.
```